instrumentation/runtime-telemetry/README.md

```diff
@@ -5,5 +5,6 @@
 | `otel.instrumentation.runtime-telemetry.emit-experimental-telemetry`     | Boolean | `false` | Enable the capture of experimental metrics.                       |
 | `otel.instrumentation.runtime-telemetry-java17.enable-all`               | Boolean | `false` | Enable the capture of all JFR based metrics.                      |
 | `otel.instrumentation.runtime-telemetry-java17.enabled`                  | Boolean | `false` | Enable the capture of JFR based metrics.                          |
+| `otel.instrumentation.runtime-telemetry.jvm-gc-cause-attribute-enabled`  | Boolean | `false` | Enable the capture of the jvm.gc.cause attribute within the jvm.gc.duration metric.
```

Comment:
You may need to reformat the table. I probably would call it `capture-gc-cause`.